documentation/api/metrics/v1/changes-from-puppetdb-v3.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "PuppetDB 5.2 » Metrics API » v1 » Upgrading to PuppetDB 4 or newer"
+title: "Metrics API v1"
 layout: default
```

Comment:
Wondered if this should keep the upgrade context, say "Metrics API changes in version 4" or something.